tests/test_imports.py

```diff
@@ -0,0 +1,93 @@
+"""Tests for module imports."""
+
+import importlib
+import os
+import pkgutil
+import subprocess
+import sys
+from collections.abc import Sequence
+
+import pytest
+from pytest import param
+
+pytestmark = pytest.mark.skipif(
```

Comment:
this is not used anywhere or? Id guess the intent is to use it like a `@pytestmark` before the tests below, or is this just here to skip the entire file here somehow?

Reply:
This is in fact `pytest` magic to indicate the latter: https://docs.pytest.org/en/7.1.x/example/markers.html#marking-whole-classes-or-modules

As pointed out to Alex, at some appropriate point I want to improve our optional dependency loading a bit and then this mark might become irrelevant, but for now I think we need it. Please resolve if OK